bbot/modules/smuggler.py

```diff
@@ -37,7 +37,7 @@ def handle_event(self, event):
             self.scanned_hosts.add(host_hash)
 
         command = [
-            "python",
+            "python3",
```

Comment:
Good catch although this should probably be `sys.executable` instead of `"python3"` to ensure it runs on the same python version as BBOT.

```suggestion
            sys.executable,
```